wikipedia_transform/record_transform.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod
+
+
+class RecordTransform(ABC):
+    """An interface to transform Records into embeddings."""
```

Comment:
Extractor: input outside the current process -> output in the current process
```
() -> T
```

Transformer: input in the current process -> output in the current process

```
T -> U
```

Loader: input in the current process -> output outside the current process
```
U -> None
```